src/pages/components/tabs/usage.mdx

```diff
@@ -402,11 +508,13 @@ label.
 <Row>
 <Column colLg={8}>
 
-![Examples of tabs with icons](images/tab-usage-12.png)
+![Examples of contained tabs with icons](images/tab-usage-15.png)
 
 </Column>
 </Row>
 
+<Caption>Example of contained tabs with icons.</Caption>
+
 ### Icon-only tabs
```

Comment:
In this "Icon-only tabs" section, it just needs to add the word "focus" to this sentence:

"Always use a tooltip for an icon description on hover and focus to add clarity."